evaluator/autoware_control_evaluator/src/control_evaluator_node.cpp

```diff
@@ -186,6 +186,53 @@ DiagnosticStatus ControlEvaluatorNode::generateYawDeviationDiagnosticStatus(
   return status;
 }
 
+DiagnosticStatus ControlEvaluatorNode::generateGoalLongitudinalDeviationDiagnosticStatus(
+  const Pose & ego_pose)
+{
+  DiagnosticStatus status;
+  const double long_goal =
+    metrics::calcLongitudinalDeviation(ego_pose, route_handler_.getGoalPose().position);
```

Comment:
we want to caluclate distance from goal_base to ego_pose, so I think the first arg should be goal.
same to other functions

```
/**
 * @brief calculate longitudinal deviation of the given ego pose from the modified goal pose
 * @param [in] base_pose base pose
 * @param [in] target_point target point
 * @return calculated statistics
 */
Stat<double> calcLongitudinalDeviation(const Pose & base_pose, const Point & target_point);

```

Reply:
That's for sure !
I have replaced the argument!